pom.xml

```diff
@@ -119,28 +119,6 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>3.1.2</version>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-gpg-plugin</artifactId>
-                <version>3.1.0</version>
-                <executions>
-                    <execution>
-                        <id>sign-artifacts</id>
-                        <phase>verify</phase>
-                        <goals>
-                            <goal>sign</goal>
-                        </goals>
-                        <configuration>
-                            <keyname>${gpg_keyname}</keyname>
-                            <passphraseServerId>${gpg_passphrase}</passphraseServerId>
-                            <gpgArguments>
-                                <arg>--pinentry-mode</arg>
-                                <arg>loopback</arg>
-                            </gpgArguments>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
```

Comment:
Any reason for disabling the plugin?

Reply:
the gpg plugin part was something adam and i discussed where it was in pom.xml but he never used it when building / publishing the plugins. I removed it because it required additional setup that i thought was unnecessary.